addons/base_import/models/base_import.py

```diff
@@ -1365,10 +1362,10 @@ def _import_image_by_url(self, url, session, field, line_number):
         :return: the replacement value
         :rtype: bytes
         """
-        maxsize = int(config.get("import_image_maxbytes", DEFAULT_IMAGE_MAXBYTES))
-        _logger.debug("Trying to import image from URL: %s into field %s, at line %s" % (url, field, line_number))
+        maxsize = int(config.get("import_file_maxbytes", config.get("import_image_maxbytes", DEFAULT_FILE_MAXBYTES)))
+        _logger.debug("Trying to import file from URL: %s into field %s, at line %s", url, field, line_number)
         try:
-            response = session.get(url, timeout=int(config.get("import_image_timeout", DEFAULT_IMAGE_TIMEOUT)))
+            response = session.get(url, timeout=int(config.get("import_file_timeout", config.get("import_image_timeout", DEFAULT_FILE_TIMEOUT))))
```

Comment:
IMO, you can deprecated `import_image_timeout`in favor of `import_file_timeout` ?